pkg/mimir/mimir.go

```diff
@@ -108,6 +108,8 @@ type Config struct {
 	NoAuthTenant                    string                 `yaml:"no_auth_tenant" category:"advanced"`
 	ShutdownDelay                   time.Duration          `yaml:"shutdown_delay" category:"advanced"`
 	MaxSeparateMetricsGroupsPerUser int                    `yaml:"max_separate_metrics_groups_per_user" category:"experimental"`
+	MaxCostAttributionPerUser       int                    `yaml:"max_cost_attribution_per_user" category:"experimental"`
```

Comment:
This would also reduce some of the noise in the diff, since we don't need to pass the value down separately from the limits, which we already pass around everywhere.